### PR TITLE
Fix style in REST API usage

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | master, 4.5
| Edition       | All

Fix style in "REST API usage" pages using Ibexa and Microsoft `vale` styles.

Fixed:
* [Ibexa.FutureTense] Use present tense instead of future.
* [Ibexa.Semicolon] Only use the semicolon if you're absolutely sure what you're doing.
    - https://learn.microsoft.com/en-us/style-guide/punctuation/semicolons
    - https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation
* [Microsoft.Avoid] Don't use 'and so on'. See the A-Z word list for details.
    - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/and-so-on
* [Microsoft.Wordiness] Consider using 'to' instead of 'In order to'.
* [Microsoft.Wordiness] Consider using 'most' instead of 'Almost all'.
* [Microsoft.Contractions] Use 'it's' instead of 'It is'.
* [Microsoft.Contractions] Use 'isn't' instead of 'is not'.
* [Microsoft.Contractions] Use 'hasn't' instead of 'has not'.
* [Microsoft.Contractions] Use 'they're' instead of 'they are'.
* [Microsoft.Contractions] Use 'doesn't' instead of 'does not'.
* [Microsoft.Contractions] Use 'shouldn't' instead of 'should not'.

Ignored:
* docs/api/rest_api/rest_api_usage/rest_api_usage.md
    - 67:28 error [Microsoft.Dashes] Remove the spaces around ' —'.
    - 68:30 error [Microsoft.Dashes] Remove the spaces around ' —'.
    - 69:39 error [Microsoft.Dashes] Remove the spaces around ' —'. (used a lot surrounded with spaces in the whole docs, and I disagree)
* docs/api/rest_api/rest_api_usage/rest_responses.md
    - 98:26  warning [Microsoft.Adverbs] Consider removing 'generally'. (got to check when exactly response got a Location header, kept for another time)
* docs/api/rest_api/rest_api_usage/rest_requests.md
    - 232:9 warning [Ibexa.Semicolon] (legit usage)
    - 295:10 warning [Ibexa.Semicolon] (legit usage)

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/api/rest_api/rest_api_usage/rest_requests.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ For method action details per resource, see the [REST API reference](../rest_api
 
     If applicable, both methods are always mentioned in the specifications.
 
-Unsafe methods will require a CSRF token if [session-based authentication](rest_api_authentication.md#session-based-authentication) is used.
+Unsafe methods require a CSRF token if [session-based authentication](rest_api_authentication.md#session-based-authentication) is used.
 
 ### OPTIONS method
 
-Any URI resource that the REST API responds to will respond to an `OPTIONS` request.
+Any URI resource that the REST API responds to also responds to an `OPTIONS` request.
```
Comment (MagdalenaZuba) [inline docs/api/rest_api/rest_api_usage/rest_requests.md:48]:
```suggestion
Any URI resource that the REST API responds to an `OPTIONS` request.
```

**Review by MagdalenaZuba — APPROVED** [review]

File: docs/api/rest_api/rest_api_usage/rest_requests.md (hunk 1 above)
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_usage/rest_requests.md:48]: @MagdalenaZuba
```suggestion
Any URI resource of the REST API responds to the `OPTIONS` method.
```

Comment (MagdalenaZuba) [reply docs/api/rest_api/rest_api_usage/rest_requests.md:48]:
```suggestion
Any URI resource used by the REST API responds to an `OPTIONS` request.
```

Comment (adriendupuis) [reply docs/api/rest_api/rest_api_usage/rest_requests.md:48]:
```suggestion
Any REST API URI responds to an `OPTIONS` request.
```

**Pull request merged**